src/extensions/insights/insights.cmd.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Color, Box, Text } from 'ink';
+import { Command, CLIArgs } from '../cli';
+import { Flags } from '../paper/command';
+import { InsightManager } from './insight-manager';
+import { InsightResult } from './insight';
+
+export default class InsightsCmd implements Command {
+  name: string;
+  description: string;
+  group: string;
+  opts: string[][];
+  insightManager: InsightManager;
+  constructor(insightManager: InsightManager) {
+    this.insightManager = insightManager;
+    this.opts = [['l', 'list', 'list all insights']];
+    this.name = 'insights [...names]';
+    this.description = 'start a dev environment for a workspace or a specific component';
+    this.group = 'development';
+  }
+
+  async render([names]: CLIArgs, { list }: Flags) {
+    if (list) {
+      //return <Color green>There are many insights</Color>
+      const results = this.insightManager.listInsights();
+      const listItems = results.map(insight => (insight += '\n'));
+      return <Color blueBright>{listItems}</Color>;
+    }
+    if (names) {
+      let results: InsightResult[] = [];
+      if (Array.isArray(names)) {
+        results = await this.insightManager.run(names);
+      } else {
+        results = await this.insightManager.run([names]);
+      }
+      return <Color blueBright>{results}</Color>;
+    } else {
```

Comment:
remove the else, you don't need else after return

Reply:
Acutally you will need it once you won't return here but rather take the results into the react component